### PR TITLE
Ethereal Lungs

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

Ethereals now have special lungs that can breathe water and electrolyze it! Ported from tgstation/tgstation#62197

# Why it's good for the game

Thought it would be funny, ethereals could use some more unique stuff.

# Spriting
image

# Wiki Documentation
Mention on the ethereal wiki page they can breathe water as a substitute for oxygen and doing so converts it into hydrogen and oxygen.

# Changelog

:cl:  
rscadd: ethereals have electrolyzer breath now
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
Water to EXPLOSION maker

**Comment by iamthesilly** [comment]
> Water to EXPLOSION maker

With how tiny the gas amounts used in breathing are I don't think it will be a problem

**Comment by Yarinoi** [comment]
ethereal hydrogen generator
good change

**Comment by iloveloopers** [comment]
How many do you need to match an electrolyzer

**Comment by ju45he** [comment]
Etheral janitor using water vapor for mass cleaning (gone wrong)

**Review by ToasterBiome — APPROVED** [review]

File: code/modules/research/xenobiology/crossbreeding/_structures.dm [hunk 1]
```diff
@@ -424,7 +424,7 @@ GLOBAL_LIST_EMPTY(bluespace_slime_crystals)
...
+	var/type = pick(/obj/item/reagent_containers/food/snacks/meat/slab,/obj/item/organ/heart,/obj/item/organ/heart/freedom,/obj/item/organ/lungs,/obj/item/organ/lungs/plasmaman,/obj/item/organ/lungs/ethereal,/obj/item/organ/lungs/slime,/obj/item/organ/liver,/obj/item/organ/liver/plasmaman,/obj/item/organ/liver/alien,/obj/item/organ/eyes,/obj/item/organ/eyes/night_vision/alien,/obj/item/organ/eyes/night_vision,/obj/item/organ/eyes/night_vision/mushroom,/obj/item/organ/tongue,/obj/item/organ/stomach,/obj/item/organ/stomach/plasmaman,/obj/item/organ/stomach/ethereal,/obj/item/organ/ears,/obj/item/organ/ears/cat,/obj/item/organ/ears/penguin)
```
Comment (Moltijoe) [inline code/modules/research/xenobiology/crossbreeding/_structures.dm:427]: why is it not just picking from subtypes of obj/item/organ/lung?

Comment (iamthesilly) [reply code/modules/research/xenobiology/crossbreeding/_structures.dm:427]: Because then it would pick non-edible ones like cybernetics

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**